### PR TITLE
Feature: Experimental Bybit Change

### PR DESCRIPTION
**Pull request by MinsungKim-BlockCrafters**
## What does this PR do?

Bybit experimental changes

## Changes

- Fixed `Bybit.subscribe`
- Included `positions` as authenticated channels
- Fixed `Bybit`'s several methods to make working

### CODE REVIEW

File: cryptofeed/exchanges/bybit.py [hunk 1]
```diff
@@ -160,14 +170,21 @@ async def subscribe(self, connection: AsyncConnection, quote: str = None):
...
-                            "args": [f"{chan}.{pair}"]
-                        }
-                    ))
+                    chans.append(f"{chan}.{pair}")
                     LOG.debug(f'{connection.uuid}: Subscribing to public, quote: {quote}, {chan}.{pair}')
 
+            elif chan in self.channels_without_symbol:
+                # chans.append(f"{chan}")
+                # LOG.debug(f'{connection.uuid}: Subscribing to private, {chan}')
+                pass
```
Comment (boseok-bc) [inline cryptofeed/exchanges/bybit.py]: Shouldn't it be uncommented?

File: cryptofeed/exchanges/bybit.py [hunk 2]
```diff
@@ -160,14 +170,21 @@ async def subscribe(self, connection: AsyncConnection, quote: str = None):
...
+
+            else:
+                raise NotImplementedError
+
+        for i in range(0, len(chans), 10):
+            await connection.write(json.dumps({"op": "subscribe", "args": chans[i:i+10]}))
```
Comment (boseok-bc) [inline cryptofeed/exchanges/bybit.py:177]: 왜 이렇게 connection 만드는거에요?

File: cryptofeed/exchanges/bybit.py (hunk 1 above)
Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bybit.py]: I thought so, but it's already subscribed in line 155.

Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bybit.py]: I will do some experiments to confirm this.

File: cryptofeed/exchanges/bybit.py (hunk 2 above)
Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bybit.py:177]: This is the same way as original author made subscription. (Each batch holds 10 subscriptions)

Reference: https://github.com/block-crafters/cryptofeed/blob/2c4dcb4b87cacf526e6cee36e65ba989226e7f2b/cryptofeed/exchanges/bitmex.py#L872-L874

File: cryptofeed/exchanges/bybit.py (hunk 1 above)
Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bybit.py]: Yes, it's already subscripted by line 155. (I was little bit confused here) I will remove `elif` and `else` control flow.

File: cryptofeed/exchanges/bybit.py (hunk 2 above)
Comment (boseok-bc) [reply cryptofeed/exchanges/bybit.py:177]: 아 적당히 나눠서 connection 만드는 거군요 감사합니다.

File: cryptofeed/exchanges/bybit.py (hunk 1 above)
Comment (boseok-bc) [reply cryptofeed/exchanges/bybit.py]: ~~아하.. 이미 쓰여져있는 코드자체가 조금 완성도가 떨어져보이네요
https://github.com/block-crafters/cryptofeed/pull/62/files/488e7c0ad9abebc28b3c34d2ae71bb5b53ec2d69#diff-d7ca79575de6396d588ad3f61bdd383d160732c8ec48c80b5f3812a45d646618R148-R158
여기도 그렇고요~~

Comment (boseok-bc) [reply cryptofeed/exchanges/bybit.py]: 다시 코드를 좀 읽어보니깐 https://github.com/block-crafters/cryptofeed/pull/62/files#diff-d7ca79575de6396d588ad3f61bdd383d160732c8ec48c80b5f3812a45d646618R168-R178
여기에 해당하는 코드가 "USDT" public channel을 수신하는 부분이라서 테스트시에 BTCUSD, ETHUSD 등의 Inverse perpetual로 했다면 여기 코드와 무관하게 subscription 되겠군요.

**Review by boseok-bc — APPROVED** [review]

File: cryptofeed/exchanges/bybit.py [hunk 3]
```diff
@@ -446,12 +453,10 @@ async def _order(self, msg: dict, timestamp: float):
                 'create_type': data.get('create_type'),
             }
             LOG.debug(f"Handle order msg: {data_to_log}")
-            await self.callback(ORDER_INFO, feed=self.id, symbol=symbol, data=data, receipt_timestamp=timestamp)
+            await self.callback(ORDER_INFO, feed=self.id, receipt_timestamp=timestamp, **data)
 
     def parse_balance(self, data: dict[str, Any]) -> dict[str, Any]:
-        balance: dict[str, Any] = {
-            'timestamp': self.timestamp_normalize(data['timestamp'])
```
Comment (KunduzB) [inline cryptofeed/exchanges/bybit.py:453]: Don't we need timestamp for balance data ?

Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bybit.py:453]: Look at line 487.

**Review by KunduzB — APPROVED** [review]

**Pull request merged**